src/components/Form/Form.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Form.scss';
+import authRequests from '../../Helpers/Data/authRequests';
+
+const defaultEvent = {
+  uid: '',
+  event: '',
+  startdate: '',
+  location: '',
+};
+
+class Form extends React.Component {
+  static propTypes = {
+    onSubmit: PropTypes.func,
+  }
+
+  state = {
+    newEvent: defaultEvent,
+  }
+
+  formFieldStringState = (name, e) => {
+    e.preventDefault();
+    const tempEvent = { ...this.state.newEvent };
+    tempEvent[name] = e.target.value;
+    this.setState({ newEvent: tempEvent });
+  }
+
+  eventChange = e => this.formFieldStringState('event', e);
+
+  dateChange = e => this.formFieldStringState('startdate', e);
+
+  locationChange = e => this.formFieldStringState('location', e);
+
+  formSubmit = (e) => {
+    e.preventDefault();
+    const { onSubmit } = this.props;
+    const myEvent = { ...this.state.newEvent };
+    myEvent.uid = authRequests.getCurrentUid();
+    onSubmit(myEvent);
+    this.setState({ newEvent: defaultEvent });
+  }
+
+  render() {
+    const { newEvent } = this.state;
+    const title = () => <h2>Add New Event:</h2>;
+    return (
+      <div className="eventForm col">
+      {title()}
+        <form onSubmit={this.formSubmit}>
+          <div className="form-group">
+            <label htmlFor="event">Event</label>
+            <input
+              type="text"
+              className="form-control"
+              id="event"
+              placeholder="Ayden's Baseball Game"
+              value={newEvent.event}
+              onChange={this.eventChange}
+            />
+          </div>
+          <div className="form-group">
+            <label htmlFor="startDate">Start Date</label>
+            <input
+              type="date"
+              className="form-control"
+              id="startDate"
+              placeholder="02/23/2019"
+              value={newEvent.startdate}
```

Comment:
Make sure to check this value against the existing data (casing):
https://github.com/acumberlander/react-nutshell/blob/master/src/db/events.json#L5